src/main.ts

```diff
@@ -1,10 +1,21 @@
 import { createApp } from "vue";
+import * as Sentry from "@sentry/vue";
+import { Integrations } from "@sentry/tracing";
 import App from "./App.vue";
 import router from "./router";
 import store from "./store";
 import "bootstrap";
 import "bootstrap/dist/css/bootstrap.min.css";
 
+Sentry.init({
```

Comment:
Are you sure that this should be added here, and not in App.vue? (I'm not sure, the documentation says "Passing in `Vue` is optional, but if you do not pass it `window.Vue` must be present." and I'm in doubt whether `window.Vue` is present here)